frontend/src/views/admin/offertable/index.tsx

```diff
@@ -161,56 +165,55 @@ export function ConnectedOfferTable(props) {
 
         return [
             {
-                Header: "Last Name",
+                Header: generateHeaderCell("Last Name"),
                 accessor: "applicant.last_name",
                 Cell: generateApplicantCell("last_name"),
             },
             {
-                Header: "First Name",
+                Header: generateHeaderCell("First Name"),
                 accessor: "applicant.first_name",
                 Cell: generateApplicantCell("first_name"),
             },
             {
-                Header: "Email",
+                Header: generateHeaderCell("Email"),
                 accessor: "applicant.email",
                 Cell: generateApplicantCell("email"),
             },
             {
-                Header: "Position",
+                Header: generateHeaderCell("Position"),
                 accessor: "position.position_code",
             },
             {
-                Header: "Hours",
+                Header: generateHeaderCell("Hours"),
                 accessor: "hours",
                 className: "number-cell",
                 maxWidth: 70,
                 Cell: generateAssignmentCell("hours"),
             },
             {
-                Header: "Status",
+                Header: generateHeaderCell("Status"),
                 id: "status",
                 // We want items with no active offer to appear at the end of the list
                 // when sorted, so we set their accessor to null (the accessor is used by react table
                 // when sorting items).
-                accessor: (data) =>
+                accessor: (data: { active_offer_status: string }) =>
```

Comment:
`data: Assignment`

Reply:
```
TS2367: This condition will always return 'false' since the types '"accepted" | "rejected" | "withdrawn" | "provisional" | "pending" | "no_offer" | null' and '"No Contract"' have no overlap.
```
The Assignment type lists all possible values for active_offer_status; "No Contract" is not one of them, thus TS complains about the condition check below always being false

Reply:
I see...does a new type get created for the data? If so, we should use that type.

Reply:
I am unsure about the exact logic here. Under what circumstances would "No Contract" be set as a status value for an assignment? I thought before that No Contract was a nice way of displaying null in the table to the users, and thus it should never be the actual status value in an Assignment object returned by the backend.